python/lsst/pipe/analysis/visitAnalysis.py

```diff
@@ -716,19 +796,68 @@ def calibrateCatalogs(self, dataRef, catalog, fluxMag0, photoCalibDataset):
             self.zpLabel = "FLUXMAG0"
             calibrated = calibrateSourceCatalog(catalog, self.zp)
 
+        if self.config.doApplyExternalSkyWcs:
+            wcs = dataRef.get(repoInfo.skyWcsDataset)
+            for record in catalog:
+                record.updateCoord(wcs)
+            if "wcs" not in self.zpLabel:
+                self.zpLabel += "\nwcs: " + self.config.externalSkyWcsName.upper()
+
         return calibrated
 
 
 class CompareVisitAnalysisConfig(VisitAnalysisConfig):
-    doApplyUberCal1 = Field(dtype=bool, default=True, doc="Apply uberCal (jointcal/meas_mosaic) results " +
-                            "to input1? FLUXMAG0 zeropoint from SFM is applied if doApplyUberCal is False")
+    doApplyExternalPhotoCalib1 = Field(dtype=bool, default=True,
+                                       doc=("Whether to apply external photometric calibration (e.g. "
+                                            "fgcmcal/jointcal/meas_mosaic) via an "
+                                            "`lsst.afw.image.PhotoCalib` object to input1.  If `True`, "
+                                            "uses ``externalPhotoCalibName1`` field to determine which "
+                                            "calibration to load.  If `False`, the instrumental flux "
+                                            "corresponding to 0th magnitude, fluxMag0, from SFM is "
+                                            "applied."))
```

Comment:
I was wondering if there was an easier way of doing this with less copy/pasting by making a sub-config and having it linked to twice, but it probably isn't worth it at this point.  But something to consider.

Reply:
I also consider this every time I make such updates...not having an immediate solution come to mind, I also concluded it's not worth it at this point.